public/forms/validation/form.ttl

```diff
@@ -85,7 +85,7 @@ ext:validationsFG a form:FieldGroup ;
 fields:validatonSelectorF a form:Field ;
   sh:name "Validation type" ;
   sh:order 2 ;
-  sh:path rdf:type ;
+  sh:path rdf:type, form:grouping ;
```

Comment:
Oh, I see what you were asking before. Since we're in custom component space we can just make assumptions in the component itself, so this wasn't strictly needed. But fine to keep if it works 👍.

Reply:
Perfect! Just found out that it works like this :D